ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/page/v1/page/customheaderlibs.html

```diff
@@ -18,6 +18,11 @@
      data-sly-test.clientLibRef="${formstructparser.clientLibRefFromFormContainer}">
     <sly data-sly-test="${clientLibRef}" data-sly-call="${clientlib.css @ categories=clientLibRef}"/>
 </sly>
+<sly data-sly-use.clientlib="core/wcm/components/commons/v1/templates/clientlib.html"
+     data-sly-use.formstructparser="com.adobe.cq.forms.core.components.models.form.FormStructureParser"
```

Comment:
how would it work for multiple form container in a page?

Reply:
Do we even need to have different themes in multiple form containers inside a page?
Also, even in master we are applying theme via page component itself through slingcontext aware config

Also, there is no option in site editor to have a different theme for form other than site.

Reply:
cc @pankaj-parashar

Reply:
cc @sakshi-arora1 @pankaj-parashar